package-lock.json

```diff
@@ -4109,7 +4118,8 @@
         "ansi-regex": {
           "version": "2.1.1",
           "bundled": true,
-          "dev": true
+          "dev": true,
+          "optional": true
```

Comment:
The intent was simply to fix the specific eslint-disable-line comments that were causing problems with the reformat, not to remove/replace all of them.  I guess I don't see the issue with having them be fixed up if, at some point in the future, and edit happens to cause them to break.

Reply:
Generally, node@8.16.0 comes with npm 6.9, which makes me wonder how you found the configuration that you have.  I'm assuming you're suggestion that I use the configuration that you describe to avoid adding these option lines?

$#@@$#$ npm versioning